hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/HederaWorldState.java

```diff
@@ -88,14 +91,16 @@ public HederaWorldState(
             final CodeCache codeCache,
             final SigImpactHistorian sigImpactHistorian,
             final GlobalDynamicProperties dynamicProperties,
-            final @HandleThrottle FunctionalityThrottling handleThrottling) {
+            final @HandleThrottle FunctionalityThrottling handleThrottling,
+            @NonNull final RecordsHistorian recordsHistorian) {
```

Comment:
(and here we have _both_ orders annotation/`final` and `final`/annotation - is there a rule I should know?)